### ISSUE TITLE
Clip model sample training code

### ISSUE THREAD

**Issue by ShahabMokari** [facebookresearch/multimodal#383]
### 🚀 The feature, motivation and pitch

Hello
I wonder if you are going to have sample training code (like the ones you have in  "/examples" folder) for CLIP model?

**Comment by pikapecan** [comment]
Yup! We are going to provide training examples for CLIP soon, @pbontrager is helping with it.

Peng

**Comment by pikapecan** [comment]
Hi, 
we currently haven't provided training examples for CLIP and it's not on our near future plan. Plz feel free to refer to other examples for your case.

**Comment by ankitade** [comment]
Hi @ShahabMokari, are you looking for a tutorial to train clip or a full pretraining script

**Comment by ShahabMokari** [comment]
Sorry for late reply.
@ankitade  Yes I am looking for tutorial on training a CLIP model from scratch (based on a custom dataset)
